src/tests/Microsoft.Diagnostics.Monitoring.EventPipe/Microsoft.Diagnostics.Monitoring.EventPipe.UnitTests.csproj

```diff
@@ -8,7 +8,9 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\Microsoft.Diagnostics.Monitoring.EventPipe\Microsoft.Diagnostics.Monitoring.EventPipe.csproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\Microsoft.Diagnostics.NETCore.Client\Microsoft.Diagnostics.NETCore.Client.csproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\Microsoft.Diagnostics.NETCore.Client\Microsoft.Diagnostics.NETCore.Client.UnitTests.csproj" />
-    <ProjectReference Include="..\EventPipeTracee\EventPipeTracee.csproj" PrivateAssets="all" />
+    <ProjectReference Include="..\EventPipeTracee\EventPipeTracee.csproj" PrivateAssets="all">
```

Comment:
I found that this still causes nuget to evaluate csproj dependencies as if they were a part of the same assembly graph. Long term we may want sln level dependencies if they work in the build lab.

Reply:
Without this, there was a downgrade of Microsoft.Extensions.Logging.EventSource, however this seems to be working locally for me and on the PR builds.